### ISSUE TITLE
Indented boxes get cut off

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#1642]
When indented, the right edge of boxes seem to get cut-off

![image](https://user-images.githubusercontent.com/1673303/128591500-ddc3a20b-da81-4cc4-ba43-d3ba6719349b.png)

Example https://nus-cs2103-ay2122s1.github.io/website/admin/ip-w4.html

**Comment by jonahtanjz** [comment]
Looks like this can be fixed with some CSS

**Comment by ong6** [comment]
Hi, looks like removing the width in box-container fixes the issue, I will add fix to my PR at #1647.

```
.box-container {
        /* width: 100%; */
        padding: 0;
        border-radius: 6px;
    }
```
![image](https://user-images.githubusercontent.com/19247856/128746246-7f2a589e-566b-42e6-bced-b788e975f28a.png)
